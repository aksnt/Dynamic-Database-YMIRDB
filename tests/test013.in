SET a 1
SET b 4 5 a
SET c b a 1
SET d a b c 7
SET e     5 7   8
LIST ENTRIES
LIST SNAPSHOTS
LIST KEYS
SNAPSHOT
DROP 1
SNAPSHOT
LIST SNAPSHOTS
PURGE e
GET e
DEL d
GET d
PURGE d
BYE
