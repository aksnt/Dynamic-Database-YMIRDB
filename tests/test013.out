> ok

> ok

> ok

> ok

> ok

> e [5 7 8]
d [a b c 7]
c [b a 1]
b [4 5 a]
a [1]

> no snapshots

> e
d
c
b
a

> saved as snapshot 1

> ok
 
 > saved as snapshot 2

 > 2
 1

 > ok

 > no such key

 > ok

 > 
